modules/administration/pages/troubleshooting/tshoot-bootstrap-repo-for-eol-products.adoc

```diff
@@ -10,19 +10,32 @@ For more information about bootstrap repositories, see xref:client-configuration
 
 .Procedure: Creating Bootstrap Repositories for End-Of-Life Products
 
-. At the command prompt on the {productname} Server, as root, list the available unsupported bootstrap repositories with the [option]``--force`` option, for example:
+. At the command prompt of the {productname} container host, as root, enter the server container with:
 +
+
+----
+mgrctl term
+----
+
+. Inside the container, execute the following steps:
++
+
+--
+.. List the available unsupported bootstrap repositories with the [option]``--force`` option, for example:
++
+
 ----
 mgr-create-bootstrap-repo --list --force
 1. SLE-12-SP2-x86_64
 2. SLE-12-SP3-x86_64
 ----
-. Create the bootstrap repository, using the appropriate repository name as the product label:
+
+.. Create the bootstrap repository, using the appropriate repository name as the product label:
```

Comment:
[advanced nit] when I worked with professional editors, we were not allowed to have "bare using" because it's ambiguous (for translations, typically). In this sentence, are we creating the repository using the repository name, or is the repository using the appropriate repository name?

Instead, you can specify `by using` to have the first meaning, and `that uses` to have the second meaning.

This is very subtle, but IMHO important enough to at least mention :)

```suggestion
.. Create the bootstrap repository by using the appropriate repository name as the product label:
```

Reply:
This surely makes sense--I'm always happy to learn language details!